Challo/Challo/Modules/Review/Views/ViewComponents/ExistingReviewView.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  ExistingReviewView.swift
+//  Challo
+//
+//  Created by Tan Le Yang on 12/4/21.
+//
+
+import SwiftUI
+
+struct ExistingReviewView: View {
+
+    var review: Review
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("You have already written a review for this booking")
+                .font(.title)
+                .foregroundColor(.themeSecondary)
```

Comment:
Dont need this bah. Once the user sees the review it's implied that the review is written alr